cmd/connector/config/config.toml

```diff
@@ -0,0 +1,15 @@
+[web_socket]
+# IP with port used to recieve data via ws. Should be compatible with the one from node. See [HostDriverConfig].URL
```

Comment:
maybe add 4 white spaces on all configuration lines? I know they are not standard but help a lot whenever trying to change something or add new configs